_sass/custom/media-breakpoint.scss

```diff
@@ -24,4 +24,16 @@
     .btn-primary {
         font-size: revert !important;
     }
+}
+
+@include media-breakpoint-up(l) {
+    .container {
+        max-width: 1376px;
```

Comment:
This will be a global change that is needed for header nav, homepage, and footer content as well, besides interior pages.